python/requirements.txt

```diff
@@ -59,6 +59,8 @@ moto
 mypy
 networkx
 numba
+raydp-nightly; platform_system != "Windows"
```

Comment:
@clarkzinzow @iycheng could you provide the right place to add this for dataset tests?